.github/comment-template.md

```diff
@@ -0,0 +1,9 @@
+Sandbox environment for `{{ .branch_ref }}` has been created.
```

Comment:
I'd name the file `sandbox-comment-template.md` but I don't have a strong opinion